### ISSUE TITLE
change compact zeiton recepie

### ISSUE THREAD

**Issue by Saturnorsomthing** [amblelabs/ait#719]
just make it so it needs 1 zeiton block in each slot of the crafting table (or shard or whatever)

**Comment by Duzos** [comment]
what do mean

**Comment by MaketendoDev** [comment]
like how you make a iron block
do that with zeiton is waht im translating

**Comment by Saturnorsomthing** [comment]
yeah that since if it stays like how it is rn its bound to cause a revolution
